<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>main_user</class>
 <widget class="QMainWindow" name="main_user">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>800</width>
    <height>600</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <property name="styleSheet">
   <string notr="true"/>
  </property>
  <widget class="QWidget" name="centralwidget">
   <widget class="QLabel" name="label_bg">
    <property name="geometry">
     <rect>
      <x>0</x>
      <y>0</y>
      <width>111</width>
      <height>81</height>
     </rect>
    </property>
    <property name="text">
     <string/>
    </property>
   </widget>
   <widget class="QLabel" name="label_titre">
    <property name="geometry">
     <rect>
      <x>220</x>
      <y>60</y>
      <width>291</width>
      <height>61</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>Futura Md BT</family>
      <pointsize>22</pointsize>
     </font>
    </property>
    <property name="text">
     <string>USER MANAGEMENT</string>
    </property>
    <property name="alignment">
     <set>Qt::AlignCenter</set>
    </property>
   </widget>
   <widget class="QScrollArea" name="scrollArea">
    <property name="geometry">
     <rect>
      <x>100</x>
      <y>140</y>
      <width>511</width>
      <height>241</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">QWidget{
background-color:#00000000;
border-color: #00000000;
border: 0px;
}</string>
    </property>
    <property name="verticalScrollBarPolicy">
     <enum>Qt::ScrollBarAlwaysOff</enum>
    </property>
    <property name="horizontalScrollBarPolicy">
     <enum>Qt::ScrollBarAlwaysOff</enum>
    </property>
    <property name="widgetResizable">
     <bool>true</bool>
    </property>
    <widget class="QWidget" name="scrollAreaWidgetContents">
     <property name="geometry">
      <rect>
       <x>0</x>
       <y>0</y>
       <width>511</width>
       <height>241</height>
      </rect>
     </property>
     <widget class="QToolBox" name="toolBox">
      <property name="geometry">
       <rect>
        <x>0</x>
        <y>10</y>
        <width>491</width>
        <height>201</height>
       </rect>
      </property>
      <property name="font">
       <font>
        <family>Futura Lt BT</family>
        <pointsize>10</pointsize>
       </font>
      </property>
      <property name="styleSheet">
       <string notr="true">QToolBox {
  padding: 0px;
  border: 0px;
  border:0px solid #32414B;
}

QToolBox::selected {
  border: 2px solid #1464A0;
}

QToolBox::tab {
  background-color: #19232D;
  border: 1px solid #32414B;
  color: #F0F0F0;
  border-top-left-radius: 5%;
  border-top-right-radius: 5%;
}

QToolBox::tab:disabled {
  color: #787878;
}

QToolBox::tab:selected {
  background-color: qlineargradient(spread:pad, x1:0, y1:0.642, x2:1, y2:1, stop:0 rgba(171, 161, 148, 255), stop:1 rgba(186, 169, 128, 255));
  border-bottom: 2px solid  rgb(177, 45, 45);
}


QToolBox::tab:!selected {
  background-color: rgb(118, 111, 98);
  border-bottom: 2px solid rgb(182, 157, 80);
}

QToolBox::tab:hover:!selected {
  background-color: rgb(175, 154, 135);
  border-bottom: 2px solid rgb(255, 168, 105);
}
</string>
      </property>
      <property name="currentIndex">
       <number>0</number>
      </property>
      <widget class="QWidget" name="page">
       <property name="geometry">
        <rect>
         <x>0</x>
         <y>0</y>
         <width>491</width>
         <height>166</height>
        </rect>
       </property>
       <attribute name="label">
        <string>Page 1</string>
       </attribute>
      </widget>
     </widget>
    </widget>
   </widget>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
